packages/fiori/src/IllustratedMessage.js

```diff
@@ -0,0 +1,245 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import ResizeHandler from "@ui5/webcomponents-base/dist/delegate/ResizeHandler.js";
+import { getIllustrationDataSync } from "@ui5/webcomponents-base/dist/asset-registries/Illustration.js";
```

Comment:
Is this file "..asset-registries/Illustration.js" within the commit, somehow can't see it